src/Workspaces/CSharp/Portable/FindSymbols/CSharpDeclaredSymbolInfoFactoryService.cs

```diff
@@ -179,7 +179,7 @@ public override bool TryGetDeclaredSymbolInfo(StringTable stringTable, SyntaxNod
                         node.Kind() switch
                         {
                             SyntaxKind.ClassDeclaration => DeclaredSymbolInfoKind.Class,
-                            SyntaxKind.RecordDeclaration => DeclaredSymbolInfoKind.Record,
+                            SyntaxKind.RecordDeclaration => DeclaredSymbolInfoKind.Class,
```

Comment:
so i liek the old way this worked.  DeclaredSymbolInfoKind is roslyn specific, so it can represent Record here.  What i think we should do though i just map this to `Class` at the boundaries of NavTo.

Reply:
@CyrusNajmabadi The current approach is more simpler. Just treat records as classes (and later, record structs as structs). *Especially* that we don't need any special handling for records. But let me know if you still think otherwise.

Reply:
Yeah. I don't like it. Within roslyn itself, I think we should represent fully. Just at it boundaries would we marshall differently